### PR TITLE
feat(eap): Add EAP mutations topic

### PR DESCRIPTION
**Pull request by ayirr7**
Adds the schema, topic metadata, and an example for the new Snuba eap mutations topic

Relevant to https://github.com/getsentry/snuba/pull/6216

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,126 +1,142 @@
 # Search and storage and infra is the default owner
-* @getsentry/owners-snuba @getsentry/ops
```
Comment (untitaker) [inline CODEOWNERS:2]: this entire file got reformatted, please revert that

**Review by untitaker — APPROVED** [review]
other than 1 comment, lgtm

File: CODEOWNERS (hunk 1 above)
Comment (ayirr7) [reply CODEOWNERS:2]: Not sure why but every time I try to save it, it reverts to reformatting it. Do you know how to fix that?

Comment (untitaker) [reply CODEOWNERS:2]: turn off format-on-save in your editor

Comment (ayirr7) [reply CODEOWNERS:2]: These changes are being committed via `style(lint): Auto commit lint changes` commits (the bot? is committing them). So it seems I cannot remove those?

**Comment by untitaker** [comment]
@ayirr7 if it's really the bot reformatting the file, forget about it, just merge then. sorry for the noise

File: schemas/snuba-eap-mutations.v1.schema.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            },
+            "_sort_timestamp": {
+              "type": "integer"
+            },
+            "trace_id": {
+              "$ref": "#/definitions/UUID",
+              "description": "The trace ID is a unique identifier for a trace. It is a 16 byte hexadecimal string."
+            },
+            "span_id": {
+              "type": "integer"
```
Comment (ayirr7) [inline schemas/snuba-eap-mutations.v1.schema.json:24]: @untitaker Had some questions here -- I was comparing against the spans topic schema, and I think `span_id` is a String? I noticed a couple of other examples like this too

Comment (untitaker) [reply schemas/snuba-eap-mutations.v1.schema.json:24]: you are right. please change this in the schema. it should not be an int at all, that's a mistake in the code

**Review by untitaker — APPROVED** [review]

**Pull request merged**